templates_ark/example_neighborhood_analysis_script.ipynb

```diff
@@ -1,18 +1,32 @@
 {
```

Comment:
<p>Same thing here, can you include an example output?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/angelolab/ark-analysis/pull/714/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='22'/>

Reply:
Done